_data/DeepanshuJain/DeepanshuJain.md

```diff
@@ -0,0 +1,5 @@
+name: Deepanshu Jain
+institution: DIT University 🚩
+profile_pic: DeepanshuJain.png
+quote: You can code it
+github_user: DEEPANSHUJ2610
```

Comment:
Although the username isn't case-sensitive, it doesn't hurt to make it proper :grin: 
You can do this by changing `DEEPANSHUJ2610` to `Deepanshuj2610`.